src/main/tray.ts

```diff
@@ -233,13 +239,21 @@ export class Tray {
       logo = this.trayIconSet.error;
     }
 
-    const stateMenu = this.contextMenuItems.find(item => item.id === 'state');
+    const [stateMenu, containerRuntimeMenu] = this.contextMenuItems
+      .filter(item => item.id === 'state' || item.id === 'container-runtime');
 
     if (stateMenu) {
       stateMenu.label = labels[this.kubernetesState] || labels[State.ERROR];
       stateMenu.icon = icon;
     }
 
+    if (containerRuntimeMenu) {
```

Comment:
How is this value ever false?

Reply:
`find()` can return `undefined` if no results are found. We have to make some sort of assertion before trying to access properties for `containerRuntimeMenu`.

Reply:
Right, it's a typescript thing. Because both those items exist in `contextMenuItems` and we never modify that list.